pinot-core/src/main/java/com/linkedin/pinot/core/segment/DefaultSegmentNameGenerator.java

```diff
@@ -56,6 +51,23 @@ public DefaultSegmentNameGenerator(String timeColumnName, String tableName, Stri
     _sequenceId = sequenceId;
   }
 
+  /**
+   *
+   * The sequenceId is used for numbering segments while the postfix is used to append a string to the segment name.
+   *
+   * Some examples:
+   *
+   * If the time column name exists, _segmentNamePostfix = "postfix", and _sequenceId = 1, the segment name would be
+   * tableName_minDate_maxDate_postfix_1
+   *
+   * If there is no time column, _segmentNamePostfix = "postfix" and _sequenceId = 1, the segment name would be
+   * tableName_postfix_1
+   *
```

Comment:
also, you should set postfix to "" if it is null? Otherwise you may end up getting "null" in the name? Just verify please.
add a test in any case. thanks